src/components/Section/Section.test.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+
+import { renderWithIntl } from '@folio/stripes-data-transfer-components/test/jest/helpers';
+import '../../../test/jest/__mock__';
+import { fireEvent } from '@testing-library/dom';
```

Comment:
Import `fireEvent` from `@testing-library/react`, please
And put this import under `import React from 'react'`